### PR TITLE
es6-promises -Zach Giffen

### PR DESCRIPTION
**Pull request by Zach-Giffen**
image

### CODE REVIEW

File: es6-promises/main.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { takeAChance } from './take-a-chance.js';
+const promise = takeAChance('Zach');
+promise.then((res) => console.log(res)).catch((err) => console.log(err));
```
Comment (robgmerrill) [inline es6-promises/main.js:3]: please log `err.message` to match the gif example.

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Small update to use the .message property on the err object.

**Comment by Zach-Giffen** [comment]
image

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**